tools/third_party.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+# This script contains helper functions to work with the third_party subrepo.
```

Comment:
Can this not just be in util.py ?

Reply:
I put it here because util.py contains generic utilities, whereas this script is full of deno-third-party specific stuff. I think it's cleaner.

Reply:
I put it here because util.py contains generic utilities, whereas this script is full of deno-third-party specific stuff. I think it's cleaner.